src/main/java/world/anhgelus/molehunt/Game.java

```diff
@@ -21,6 +21,8 @@ public class Game {
     public final int defaultTime = Molehunt.CONFIG.getGameDuration()*60;
     private int remaining = defaultTime;
 
+    private Timer borderTimeOffsetTime = new Timer();

```

Comment:
Save the TimerTask instead of creating a new Timer.

Reply:
what do you mean ?

Reply:
oh ok